.travis/pulp-smash-config.json

```diff
@@ -1,20 +1,18 @@
 {
     "pulp": {
         "auth": ["admin", "admin"],
+        "selinux enabled": false,
         "version": "3"
     },
-    "systems": [
+    "hosts": [
         {
             "hostname": "localhost",
             "roles": {
-                "amqp broker": {"service": "rabbitmq"},
-                "api": {"port": 8000, "scheme": "http"},
-                "mongod": {},
-                "pulp cli": {},
+                "api": {"port": 8000, "scheme": "http", "service": "nginx"},
```

Comment:
Technically speaking, nginx isn't being used for the server in this context (just the django dev server), but if it doesn't matter this is fine.

Reply:
It doesn't matter at this time. It'll matter when the Pulp 3 tests start restarting the web server.